### PR TITLE
feat: display basic conversation list items (AR-786)

### PR DESCRIPTION
**Pull request by trOnk12**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-786" title="AR-786" target="_blank">StoryAR-786</a>  Basic Conversation List
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

_ display a basic conversation list items
_ set ups a basic template for styling and themes

----
#### PR Post Merge Checklist for internal contributors

 - [x ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsRtl="true"
+            android:theme="@style/AppTheme.Light">
 
         <activity
-            android:name=".LauncherActivity"
-            android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.Dark"
-            android:exported="true">
+                android:name=".ui.MainActivity"
```
Comment (saleniuk) [inline app/src/main/AndroidManifest.xml:18]: I think we should separate `LauncherActivity` and `MainActivity` as the second one should probably have all the content available to the already logged in user and some actions for instance related to registration, login or custom backend links should be handled before that 🤔

File: app/src/main/kotlin/com/wire/android/ui/theme/Color.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+class WireColor {
+    companion object {
+        @Stable
+        val LightBlue = Color(0xFF0772DE)
+
+        @Stable
+        val LightGray = Color(0xFFEDEFF0)
+    }
+}
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/theme/Color.kt:26]:
```suggestion

object WireColor {
    @Stable
    val LightBlue = Color(0xFF0772DE)

    @Stable
    val LightGray = Color(0xFFEDEFF0)
}
```

**Review by vitorhugods — CHANGES_REQUESTED** [review]
Tiny change request + LauncherActivity comment from @saleniuk 

Other than that, nice work. So it begins!

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (saleniuk) [reply app/src/main/AndroidManifest.xml:18]: We can set this up later, it's not crucial right now

File: app/src/main/kotlin/com/wire/android/ui/theme/Color.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    onSecondary = Color.White,
+    background =  WireColor.LightGray
+)
+
+val WireDarkColors = darkColors(
+
+)
+
+
+object WireColor {
```
Comment (vitorhugods) [inline app/src/main/kotlin/com/wire/android/ui/theme/Color.kt:19]: Bringing Compose (and UI to `.kt` files in general) raises a lot of MagicNumber warnings 🤦🏼‍♂️.

I guess adding `@Suppress("MagicNumber")` works for now. But it's good to keep in mind we might need to find a more sustainable way to solve this.

**Review by vitorhugods — APPROVED** [review]
Damn Detekt.

**Pull request merged**